sources/bot/modules/chains/main/main_handlers_chain.py

```diff
@@ -108,7 +133,7 @@ async def start_handler(message: types.Message, state: FSMContext):
         text, _, not_registered = await MainHandlersChain._start_routine(message, state)
 
         await Registrar.bot.send_message(chat_id=message.chat.id, text=text)
-        await AuthExpectationHandlersChain().wait_registration(message, state, not_registered)
+        await MainHandlersChain._wait_registration(message, state, not_registered)
```

Comment:
You push old commits, replacing our new ones. Correct, please. Before making a pull request, you should always make sure that you have checked out the changes from the main repository.